addons/cni-cilium/cilium.yaml

```diff
@@ -119,14 +142,14 @@ data:
   cluster-name: default
```

Comment:
Do we want to put some name here, e.g. name of the cluster?

Reply:
We can do this but its not needed. IIRC this is only needed for cluster mesh deployment.